parser/importer_pytesseract.py

```diff
@@ -0,0 +1,170 @@
+# Python3
+# coding: utf-8
+#
+# Commit note of Daniela Grothe
+# This version uses pytesseract.
+# Only tested on Windows 8 with Python 3.8
+# and ImageMagick 6.8.8-10 Q8 x64
+# Seems as if either Win8-cmd or ImageMagick didn't want 
+# as many high-commas (didn't find my files, error msg such as:
+# "convert.exe: unable to open image `\Users\daniela\..."
+# ) so I removed some of them in the convert commands.
+# 
+#
+# Copyright 2015-2018
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+
+from PIL import Image
+
+BASE_PATH = os.getcwd()
+INPUT_FOLDER = os.path.join(BASE_PATH, "data\\img")
```

Comment:
These paths will work with Windows.
We could actually make them compatible with Linux, too.
And it's quite easy with pathlib.
https://medium.com/@ageitgey/python-3-quick-tip-the-easy-way-to-deal-with-file-paths-on-windows-mac-and-linux-11a072b58d5f
Would you like to give it a try? 😉